### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestions

### CODE REVIEW

File: pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    align-items: center;
+    gap: 2px;
+}
+
+label {
+    font-size: 0.7rem;
+}
+
+.input-check a {
+    color: var(--primary);
```
Comment (Sen-442b) [inline pages/login/login.css:89]: you can briefly specify the type of color in the variable name

File: pages/login/login.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    margin: 1.2rem 1rem 1.5rem 0rem;
+    background-color: var(--btn-primary);
+    color: var(--white);
+    padding: 0.7rem 1.1rem 0.7rem 1.1rem;
+    border: none;
+    border-radius: 4px;
+    font-size: 0.8rem;
+}
+.login-btn:hover{
+    cursor: pointer;
```
Comment (Sen-442b) [inline pages/login/login.css:104]: you don't need to give the cursor pointer property in hover pseudo class

**Review by Sen-442b — APPROVED** [review]
Everything else looks clean

File: index.html [hunk 3]
```diff
@@ -1,12 +1,148 @@
...
+                <a class="btn-link" href="./pages/product/product.html">
+                    EXPLORE NOW
+                </a>    
+                    
+                </button>
+            </section>
+
+            <!-- card2 -->
+            <section class="category-card">
+                <img class="category-card-img" src="./Assets/images/backpack1.jpg" />
```
Comment (ShreyashMahajan) [inline index.html:94]: try to give alt tag to image

File: pages/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+.horizontal .card-header-img img {
+    border-top-right-radius: unset;
+}
+
+.card-description {
+    width: 50%;
+}
+.product-qty{
+    margin: 0.5rem 0 0.5rem 0;
```
Comment (ShreyashMahajan) [inline pages/cart/cart.css:55]: try to use px for less than 1rem

File: pages/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+.qty-btn{
+    background-color: var(--secondary-grey);
+    border: none;    
+    border-radius: 50%;
+    padding: 0.1rem 0rem 0.1rem 0rem;
```
Comment (ShreyashMahajan) [inline pages/cart/cart.css:64]: same here

**Review by ShreyashMahajan — APPROVED** [review]
Just looks at the comments and then good to merge

**Pull request merged**